incubator/virtualcluster/pkg/syncer/resources/configmap/controller.go

```diff
@@ -60,108 +53,18 @@ func Register(
 		return
 	}
 	c.multiClusterConfigMapController = multiClusterConfigMapController
-
 	c.configMapLister = configMapInformer.Lister()
-	c.configMapSynced = configMapInformer.Informer().HasSynced
```

Comment:
we still need to wait for configmapSynced, or the configMapLister cannot work.

Reply:
Fixed